### ISSUE TITLE
hooks for all repositories

### ISSUE THREAD

**Issue by Cherrg** [jakubgarfield/Bonobo-Git-Server#654]
Hi there,
is there some way to create git hooks for every repository automatically?
e.g. after repository creation create pre-commit / post-commit hook file with content:
/path/to/extra/script/post-commit.sh [RepoName] [GroupName]

**Comment by willdean** [comment]
There's no way to do this at the moment.  I can see that if you have some kind of policy-enforcing hooks it would be good to have them automatically applied to every new repo.

**Comment by frankbret** [comment]
Seems like you want to use git "template" feature.  Hooks inside will be automatically imported to new repo when you init...

**Comment by Cherrg** [comment]
Yeah i was looking for a way to set these hooks on ui. This time I was really new to git and didn't know much about additional features of it.
